vercel.json

```diff
@@ -0,0 +1,24 @@
+{
+  "github": {
+    "autoAlias": true,
+    "enabled": true
+  },
+  "headers": [
+    {
+      "source": "/(.*)",
+      "headers": [
+        {
+          "key": "access-control-allow-origin",
+          "value": "*"
+        }
+      ]
+    }
+  ],
+  "redirects": [
+    {
+      "source": "/",
+      "destination": "https://github.com/vercel/zsh-theme",
+      "permanent": true
+    }
+  ]
+}
```

Comment:
No need to deploy this file, it can be read from github

Reply:
The intention was to have a custom domain rather than using GitHub URLs. Should I remove the Vercel deployment altogether?

Reply:
Yes, lets remove vercel.json